app/cloud.go

```diff
@@ -37,6 +37,17 @@ func (a *App) SendPaymentFailedEmail(failedPayment *model.FailedPayment) *model.
 	return nil
 }
 
+func (a *App) AdjustInProductLimits(limits *model.ProductLimits, subscription *model.Subscription) *model.AppError {
+	if limits.Teams != nil && *limits.Teams.Active > 0 {
```

Comment:
I think we need to nil check `teamLimits.Active`, too. Small repro of what would go wrong if `Active` was nil:

```go
package main

import (
	"fmt"
)

type TeamsLimits struct {
	Active *int
}

func main() {
	teamLimits := &TeamsLimits{
		Active: nil,
	}
	if 3 > *teamLimits.Active {
		fmt.Println("greater")
	} else {
		fmt.Println("leq")
	}
}
```

```
go run tmp.go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x47df5a]

goroutine 1 [running]:
main.main()
	/home/n/tmp.go:15 +0x1a
exit status 2
```

Reply:
ah I had assumed that no change in this limit would mean that the TeamsLimits struct would be nil - but yeah good to be safe anyways

Reply:
Ahh yeah that's true today. Could change if teams got more/different limits and not all of them were defined.